native/src/components/ExportEventButton.tsx

```diff
@@ -39,16 +45,39 @@ const ExportEventButton = ({ event }: ExportEventButtonType): ReactElement => {
       endDate = event.date.endDate.plus({ minutes: 1 }).toFormat("yyyy-LL-dd'T'00:00:00.000'Z'")
     }
 
+    let eventOptions: CalendarEventWritable = {
```

Comment:
i don't like this `let` and redeclaring tbh.
I personally think its nicer to use a function
```js
// declare function for getting recurrence rule or undefined
const getRecurrenceRule = (variables....): RecurrenceRule | undefined => {
if(exportAll.....)
}
// defining the options once
 const eventOptions : CalendarEventWritable = {
      startDate,
      endDate,
      allDay,
      calendarId,
      location: event.location?.fullAddress,
      description: event.excerpt, // Android
      notes: event.excerpt, // iOS
      recurrenceRule: getRecurrenceRule(vars...)
    }
```